uber/templates/groups/form.html

```diff
@@ -72,7 +72,7 @@
     <label class="col-sm-2 control-label">Tables</label>
     <div class="col-sm-6">
         <select name="tables">
-            {% int_options 0 9 group.tables %}
+            {% options c.ADMIN_TABLE_OPTS group.tables %}
         </select>
```

Comment:
Ha, +1 on not hard-coding this stuff.

Out of curiosity, I just did a ``grep int_options uber -r`` and it looks like there's only one other place where we hard-code our ``int_options`` these days, and that's where we have a hard-coded limit of 50 "slots" for a job.  Seems like it would be easy to make that configurable as well, but probably not worth the effort, and definitely out of scope for this PR.